src/notice/models.py

```diff
@@ -0,0 +1,14 @@
+from django.db import models
+
+
```

Comment:
notice ta kon school and kon class er jonno seta kivabe pawa jabe??

Reply:
eta ki konu ekta school er particular class er jonno naki full school er jonno seta ??